### PR TITLE
offline-split-passphrases

### PR DESCRIPTION
**Pull request by mrose17**
split files (e.g., those containing recovery keys)

### CODE REVIEW

**Comment by diracdeltas** [comment]
cc @riastradh-brave

File: cli.js [hunk 1]
```diff
@@ -524,6 +561,7 @@ switch (process.title) {
     console.log('  online-create-transaction --wallet input-file --payments input-file [--unsignedTx output-file] authopts...')
     console.log('  offline-sign-transaction --unsignedTx input-file [--signedTx output-file] [--keychains input-file]')
     console.log('  online-submit-transaction --signedTx input-file [--submitTx output-file] authopts...')
+    console.log('  offline-recover-passphrases files...')
```
Comment (riastradh-brave) [inline cli.js]: Missing offline-split-passphrases?

**Review by riastradh-brave — APPROVED** [review]
One minor nit, otherwise this looks fine to me in the context of what this is here to do.

**Review by riastradh-brave — APPROVED** [review]

File: cli.js (hunk 1 above)
Comment (mrose17) [reply cli.js]: yep. i'll add it.

**Review by NejcZdovc — APPROVED** [review]
last change looks good to me

**Pull request merged**